app/build.gradle

```diff
@@ -25,6 +25,7 @@ task conanProfile {
 task conanInstall {
     ["armv7", "armv8", "x86", "x86_64"].each { String arch ->
         exec {
+            // replace with local path to conan-binary
```

Comment:
Add conan to your $PATH, and then you won't have to do any path replacing in gradle files

Reply:
you're right, thanks!